release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3351,6 +3351,72 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.16.12
+[id="ocp-4-16-12_{context}"]
+=== RHSA-2024:6632 - {product-title} {product-version}.12 bug fix update
+
+Issued: 17 September 2024
+
+{product-title} release {product-version}.11 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6632[RHSA-2024:6632] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:6635[RHBA-2024:6635] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.12 --pullspecs
+----
+
+[id="ocp-4-16-12-enhancements_{context}"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-16-12-https-support-TransferProtocolTypes"]
+===== Supporting HTTPS for TransferProtocolTypes in Redfish APIs
+
+* TLS can be enabled for communication between ironic and the baseboard management controller (BMC) in the bootstrap phase of the install process by adding 'disableVirtualMediaTLS: false' to the Provisioning CR file created on disk by the installer. (link:https://issues.redhat.com/browse/OCPBUGS-39468[*OCPBUGS-39468*])
+
+[id="ocp-4-16-12-kubernetes-1-29-8"]
+===== Updating to Kubernetes version 1.29.8
+
+* This release contains the changes that come from the update to Kubernetes version 1.29.8. (link:https://issues.redhat.com/browse/OCPBUGS-39015[*OCPBUGS-39015*])
+
+[id="ocp-4-16-12-redirecting-git-advanced"]
+===== Redirecting on web console with Edit Source Code
+
+* There are 2 option in the _Git Advanced_ section of the web console: one option is to add a branch, tag, or commit ID and other option is to add the context directory. With this release, if you add the context directory of a particular branch, tag, or commit ID, you are redirected to that directory by selecting the *Edit source code* icon. If a branch, tag, or commit ID is not entered, you are redirected to the base url as previously expected. (link:https://issues.redhat.com/browse/OCPBUGS-38914[*OCPBUGS-38914*])
```

Comment:
```suggestion
* There are two options in the _Git Advanced_ section of the web console: one option is to add a branch, tag, or commit ID and other option is to add the context directory. With this release, if you add the context directory of a particular branch, tag, or commit ID, you are redirected to that directory by selecting the *Edit source code* icon. If a branch, tag, or commit ID is not entered, you are redirected to the base url as previously expected. (link:https://issues.redhat.com/browse/OCPBUGS-38914[*OCPBUGS-38914*])
```

Reply:
since you use "one option" and "the other", its more consistent to use "two" instead of the numeral here: https://www.ibm.com/docs/en/ibm-style?topic=measurement-numerals-versus-words

Reply:
Also option->options